Frontend/ui-driver/src/Screens/SubscriptionScreen/Handler.purs

```diff
@@ -66,6 +66,15 @@ subscriptionScreen = do
     GotoManagePlan updatedState -> do
       modifyScreenState $ SubscriptionScreenStateType (\_ -> updatedState)
       App.BackT $ App.NoBack <$> (pure $ GO_TO_MANAGE_PLAN updatedState)
+    GotoFindHelpCentre updatedState -> do
+      modifyScreenState $ SubscriptionScreenStateType (\_ -> updatedState)
```

Comment:
Try modifying the screen with this only if required, it's not mandatory to modify during exit always